pydy/viz/scene.py

```diff
@@ -24,6 +24,12 @@
     from IPython.html import widgets
     from IPython.display import display, Javascript
 
+    # Support for ipython 2 and ipython 3
+    if IPython.__version__[0] == '2':
```

Comment:
I don't think we need to make ``IPython`` a hard dependency. We can just raise a warning that IPython related functinalities will not be available.